packages/core-mobile/src/utils/screenOptions.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {
+  StackCardInterpolatedStyle,
+  StackCardInterpolationProps,
+  StackNavigationOptions,
+  TransitionPresets
+} from '@react-navigation/stack'
+import { Animated } from 'react-native'
+import Grabber from 'components/navigation/Grabber'
+import BackBarButton from 'components/navigation/BackBarButton'
+
+export const stackNavigatorScreenOptions: StackNavigationOptions = {
+  title: '',
+  headerBackTitleVisible: false,
+  headerShadowVisible: false,
+  headerTitleAlign: 'center',
+  headerBackImage: () => <BackBarButton />,
+  ...TransitionPresets.SlideFromRightIOS
+}
+
+export const modalStackNavigatorScreenOptions: StackNavigationOptions = {
+  ...stackNavigatorScreenOptions,
+  headerTitle: () => <Grabber />
+}
+
+export const modalScreensOptions: StackNavigationOptions = {
+  presentation: 'modal',
+  cardStyle: {
+    borderTopLeftRadius: 40,
+    borderTopRightRadius: 40,
+    marginTop: 75
+  },
+  gestureEnabled: true,
+  gestureDirection: 'vertical',
+  cardStyleInterpolator: forModalPresentationIOS
+}
+
+function forModalPresentationIOS({
```

Comment:
can you add a short description for this presentation? also is the `cardStyleInterpolator` prop iOS only?

Reply:
renamed to `cardStyleInterpolatorForModalPresentation`. Fyi, the name `forModalPresentationIOS` was from `react-navigation` transition presets, meaning that it was showing modals in IOS _style_. it works for both platforms.